### PR TITLE
QSA-20-03

### PR DESCRIPTION
**Pull request by stanleyshuang**
The security advisory is at https://www.qnap.com/zh-tw/security-advisory/qsa-20-03

### CODE REVIEW

File: 2020/2xxx/CVE-2020-2500.json [hunk 1]
```diff
@@ -1,18 +1,94 @@
 {
```
Comment (cve-team) [inline 2020/2xxx/CVE-2020-2500.json:1]: 'vendor_name' is a required property

Comment (cve-team) [inline 2020/2xxx/CVE-2020-2500.json:1]: 'product_name' is a required property

Comment (cve-team) [inline 2020/2xxx/CVE-2020-2500.json:1]: 'version_value' is a required property

Comment (cve-team) [inline 2020/2xxx/CVE-2020-2500.json:1]: 'url' is a required property

File: 2020/2xxx/CVE-2020-2500.json [hunk 2]
```diff
@@ -1,18 +1,105 @@
...
```
Comment (cve-team) [inline 2020/2xxx/CVE-2020-2500.json:1]: 'url' is a required property

File: 2020/2xxx/CVE-2020-2500.json (hunk 1 above)
Comment (stanleyshuang) [reply 2020/2xxx/CVE-2020-2500.json:1]: updated

Comment (stanleyshuang) [reply 2020/2xxx/CVE-2020-2500.json:1]: update

Comment (stanleyshuang) [reply 2020/2xxx/CVE-2020-2500.json:1]: updated

File: 2020/2xxx/CVE-2020-2500.json (hunk 2 above)
Comment (stanleyshuang) [reply 2020/2xxx/CVE-2020-2500.json:1]: url updated

**Comment by stanleyshuang** [comment]
close this and open the other request

**Pull request closed without merging**